frontend/javascripts/admin/job/job_list_view.tsx

```diff
@@ -157,14 +157,19 @@ function JobListView() {
     setSearchQuery(event.target.value);
   }
 
+  function getLinkToDataset(job: APIJob) {
+    if (job.datasetId != null)
+      return `/datasets/${getReadableURLPart({ name: job.datasetName || "unknown_name", id: job.datasetId })}/view`; // prefer updated link over legacy link.
+    if (job.organizationId != null && job.datasetName)
+      return `/datasets/${job.organizationId}/${job.datasetDirectoryName || job.datasetName}/view`;
+    return null;
+  }
+
   function renderDescription(__: any, job: APIJob) {
-    const linkToDataset =
-      job.datasetId != null
-        ? `/datasets/${getReadableURLPart({ name: job.datasetName || "unknown_name", id: job.datasetId })}/view` // prefer updated link over legacy link.
-        : `/datasets/${job.organizationId || ""}/${job.datasetDirectoryName || job.datasetName}/view`;
+    const linkToDataset = getLinkToDataset(job);
     if (job.type === APIJobType.CONVERT_TO_WKW && job.datasetName) {
       return <span>{`Conversion to WKW of ${job.datasetName}`}</span>;
-    } else if (job.type === APIJobType.EXPORT_TIFF && job.organizationId && job.datasetName) {
+    } else if (job.type === APIJobType.EXPORT_TIFF && linkToDataset != null) {
```

Comment:
here one could check whether the dataset name exists, maybe render the link to the annotation, and then, if job.organizationId != null, render the link to the dataset. I estimated that the chances for this are not high and would only clutter the code, so please correct me if I am wrong about that.

Reply:
yes, I think it's fine as is :+1: